manifests/b/bitvise/SSHClient/9.17/bitvise.SSHClient.installer.yaml

```diff
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.7-2-2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: bitvise.SSHClient
+PackageVersion: "9.17"
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
+InstallModes:
+- silentWithProgress
+InstallerSwitches:
+  Silent: -acceptEULA
+  SilentWithProgress: -acceptEULA
+UpgradeBehavior: install
+Protocols:
+- ssh
+FileExtensions:
+- bscp
+- tlp
+ReleaseDate: 2022-03-12
+Installers:
+- Architecture: neutral
+  InstallerUrl: https://dl.bitvise.com/BvSshClient-917.exe
+  InstallerSha256: 865156CBF63929D1E1BB47EE2B44369D3A01A8ECDF4A678256BD1B13CD324D90
+  ProductCode: '{DDE3CECB-DB63-4EAE-B202-817DD0A83B41}'
```

Comment:
```suggestion
```
Hmm, not sure why this is here? The real ProductCode is BvSshClient

Reply:
It came from running `get-wmiobject Win32_Product | Sort-Object Name | Format-Table IdentifyingNumber, Name -AutoSize` but I wasn't sure about it. Thanks for the clarification!